google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/VersionTest.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.bigtable;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/** Smoke test to ensure that release tooling doesn't accidentally corrupt the version */
+@RunWith(JUnit4.class)
+public class VersionTest {
+  @Test
+  public void testVersion() {
+    assertThat(Version.VERSION).matches("\\d+\\.\\d+\\.\\d(?:-sp.\\d+-SNAPSHOT)?");
```

Comment:
when you have a moment, can you update the master branch to include the sp suffix in the version string?

Reply:
oops, the regex needs to be:
`.matches("\\d+\\.\\d+\\.\\d(?:-sp.\\d+)(?:-SNAPSHOT)?");`

the sp suffix is independent of the snapshot